index.bs

```diff
@@ -1035,6 +1040,19 @@ the application will receive the number of streams it anticipates.
 
 </div>
 
+: <dfn for="WebTransport" method>drain()</dfn>
+:: Drains the [=WebTransport session=] associated with the WebTransport object.
+
+   When drain is called, the user agent MUST run the following steps:
+     1. Let |transport| be [=this=].
+     1. If |transport|.{{[[State]]}} is `"closed"` or `"failed"`, [=throw=] an {{InvalidStateError}}
+        and abort these steps.
+     1. Set |transport|.{{[[State]]}} to `"draining"`.
+     1. Let |draining| be |transport|.{{[[Draining]]}}.
```

Comment:
The existing definitions of [[Draining]] and concept of `draining` are now incomplete as they specifically only mention that they describe server-initiated drain. These need to be updated to include that this state can be initiated client-side.